PKGBUILD

```diff
@@ -0,0 +1,76 @@
+# Maintainer: hodasemi <michaelh.95 at t-online dot de>
+_pkgbase=breezy-desktop
+pkgname="${_pkgbase}"
+pkgver=0.1
+pkgrel=1
+pkgdesc="Breezy desktop - XR desktop"
+arch=('x86_64')
+url="https://github.com/wheaney/breezy-desktop"
+license=('GPL-3.0')
+makedepends=('ninja' 'meson' 'librsvg')
+depends=('python' 'python-pydbus' 'gnome-shell' 'XRLinuxDriver')
+conflicts=("${_pkgbase}")
+source=("git+${url}")
+md5sums=(SKIP)
+
+_uuid="breezydesktop@xronlinux.com"
+
+build() {
+    cd ${_pkgbase}
+
+    # init submodules (only required ones)
+    git submodule update --init --recursive modules/sombrero
+    git submodule update --init --recursive ui/modules/PyXRLinuxDriverIPC
+
+    # build binaries
+    cd ui
+    meson setup build
+    cd build
+    meson compile
+
+    # prepare extension
+    cd ../..
+    unlink gnome/src/schemas/com.xronlinux.BreezyDesktop.gschema.xml
+    cp ui/data/com.xronlinux.BreezyDesktop.gschema.xml gnome/src/schemas/
+    glib-compile-schemas --targetdir="gnome/src/schemas" "gnome/src/schemas"
+
+    unlink gnome/src/textures/custom_banner.png
+    cp vulkan/custom_banner.png gnome/src/textures/
+
+    unlink gnome/src/textures/calibrating.png
+    cp modules/sombrero/calibrating.png gnome/src/textures/
+
+    unlink gnome/src/IMUAdjust.frag
+    cp modules/sombrero/IMUAdjust.frag gnome/src/
+
+    # create icon
+    rsvg-convert ui/data/icons/hicolor/scalable/apps/com.xronlinux.BreezyDesktop.svg -w 64 -h 64 -o ui/data/icons/hicolor/com.xronlinux.BreezyDesktop_64.png
+    rsvg-convert ui/data/icons/hicolor/scalable/apps/com.xronlinux.BreezyDesktop.svg -w 128 -h 128 -o ui/data/icons/hicolor/com.xronlinux.BreezyDesktop_128.png
+    rsvg-convert ui/data/icons/hicolor/scalable/apps/com.xronlinux.BreezyDesktop.svg -w 256 -h 256 -o ui/data/icons/hicolor/com.xronlinux.BreezyDesktop_256.png
+    rsvg-convert ui/data/icons/hicolor/scalable/apps/com.xronlinux.BreezyDesktop.svg -w 1024 -h 1024 -o ui/data/icons/hicolor/com.xronlinux.BreezyDesktop_1024.png
```

Comment:
I converted to SVG from a low-res source PNG. I bet this upscaling looks so bad. 😂

Reply:
Actually not. I wouldn't have noticed anything really.